### PR TITLE
Issue 1 can search character

### PR DESCRIPTION
**Pull request by addy-abid**
https://user-images.githubusercontent.com/85080021/134284367-44ef460f-948e-409d-bfff-068304d4bc10.mp4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,11 +8,45 @@
...
+        </div>
+      </div>
+    </div>
   </header>
   <main>
+    <div class="container back-ground">
+      <div data-view="home">
+        <form id="home">
+          <div class="row">
+            <div class="column-full column-half">
```
Comment (codingcodymiller) [inline index.html]: Why does this have both `column-full` and `column-half`?

File: index.html [hunk 2]
```diff
@@ -8,11 +8,45 @@
...
+              <div class="search-bar">
+              <label for="search">Search a character</label> <br>
```
Comment (codingcodymiller) [inline index.html]: Please try to avoid `br` elements. You can just make this element a `block` level element.

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  console.log(xhr.status);
+  var response = xhr.response;
+  for (var i = 0; i < response.length; i++) {
+    if ($input.textContent === response.results[i].name) {
+      console.log(response.results[i].name);
+    }
+  }
+});
+xhr.send();
```
Comment (codingcodymiller) [inline js/main.js]: This doesn't look like it's actually searching for a specific character. You should be getting the data for whatever character is being searched.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by addy-abid** [comment]
https://user-images.githubusercontent.com/85080021/134411561-4389da7b-56ef-4fb4-904f-b81ac712ece2.mp4

**Comment by addy-abid** [comment]
https://user-images.githubusercontent.com/85080021/134413204-7a4114ea-5bb6-4eca-b257-87f39c00b04d.mp4

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  border-radius: 2px;
+  border: 1px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+label {
+  margin-top: 5rem;
+  margin-right: 11rem;
```
Comment (codingcodymiller) [inline css/styles.css]: This is a **_lot_** of movement. First of all, you should apply this to a class, not all `label` elements. If you have labels elsewhere in your code, this could massively break it. Secondly, this is a huge amount of margin. Why is this needed? Could you handle this with layout classes instead of pushing the element where you want it to be?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by addy-abid** [comment]
https://user-images.githubusercontent.com/85080021/134426448-2cef318f-2579-49b5-a700-1b739a98dfae.mp4

**Review by codingcodymiller — APPROVED** [review]
